web/templates/message_body.hbs

```diff
@@ -60,8 +60,11 @@
 <div class="message_edit">
     <div class="message_edit_form"></div>
 </div>
-<div class="message_expander message_length_controller tippy-zulip-delayed-tooltip" data-tooltip-template-id="message-expander-tooltip-template">{{t "[More…]" }}</div>
-<div class="message_condenser message_length_controller tippy-zulip-delayed-tooltip" data-tooltip-template-id="message-condenser-tooltip-template">[{{t "Show less" }}]</div>
+
+<div class="message_length_controller tippy-zulip-delayed-tooltip" data-tooltip-template-id="message-condenser-tooltip-template">
+    <button type="button" class="message_expander message_length_toggle" title="{{t 'Show more' }} (-)">{{t "Show more" }}</button>
+    <button type="button" class="message_condenser message_length_toggle" title="{{t 'Show less' }} (-)">{{t "Show less" }}</button>
```

Comment:
Shouldn't these buttons use different tooltip templates? (expander/condenser). We shouldn't have `title` here since we already have a tooltip for them.

Reply:
Thanks for noticing! Updated that.